torch/_inductor/ir.py

```diff
@@ -6321,17 +6321,17 @@ def codegen(self, wrapper):
         )
 
     @classmethod
-    def create(cls, x, w, b, attr, scalars, algorithm):
+    def create(cls, x, w, B, attr, scalars, algorithm):
```

Comment:
Hi @jgong5 it seems the `LinearBinary` class also need similar change (from `b` to `B`). I met an issue  [here](https://github.com/pytorch/pytorch/pull/124350#issuecomment-2130665645) and I've added this change to the `LinearBinary` class as https://github.com/pytorch/pytorch/commit/39ed894ea6944b4f24999f146484bcecb4812e1e in my PR to make CI pass there.

Reply:
Thanks for the fix. Weird that it was not exposed from previous UTs.